integration/docker_integration_test_executor.go

```diff
@@ -101,7 +118,7 @@ func givenThisContainer(t *testing.T, bundle IntegrationTestDockerExecRunnerBund
 	if err != nil {
 		t.Fatalf("Copying command wrapper to container has failed %s", err)
 	}
-	err = testRunner.Runner.RunExecutable("docker", "exec", testRunner.ContainerName, "chmod", "+x", "/piper-wrapper")
+	err = testRunner.Runner.RunExecutable("docker", "exec", "-u=root", testRunner.ContainerName, "chmod", "+x", "/piper-wrapper")
```

Comment:
Can you also specify a user id integer? If memory serves, a user with id 0 always has to exist, but it isn't necessarily named "root". (Just curious, not that I think the change is important...)

Reply:
yes, you can, I used id 1000 in the test case. not sure if `root` must exist in an image, good point to consider. I think `user = root` is more speaking compared to `user = 0`.

not sure